### DISCUSSION TITLE
Electrical Safety for Electric Vehicle Charging

### DISCUSSION THREAD

**Discussion by jc555000** [tailwindlabs/tailwindcss#6327] [General]
Electrical Safety for Electric Vehicle Charging
  ----- Magtron RCMUs short introduction

In now days, electric vehicles are more and more popular in our daily life. A data from IEA (International Energy Agency), the global stock of electric vehicle (EV) and the plug-in hybrid electric vehicle (PHEV) has reached more than 10 millions units in 2020, with the rapid global development from 2017. More and more car manufacturers pay attention to and devote themselves to the development and production of electric vehicles. IEA estimates that that by 2030, the number of the EVs and the PHEVs will exceed 125 millions, and the sales of EVs and PHEVs will reach 20% of the world's vehicle sales.
[img]https://i.loli.net/2021/11/17/BSMhiKxtuPVbqTJ.jpg[/img][/url]                                           
Bellowing is the rapid increase in the number of EV chargers. As the data shows below from IEA, the global EV chargers have grown rapidly since 2018, and the growth of charging chargers for the whole year of 2020 has exceeded 1.3 million.                                                                                <a href="https://sm.ms/image/uq9tJ2zKFyOHjfi" target="_blank"><img src="https://i.loli.net/2021/11/17/uq9tJ2zKFyOHjfi.jpg" ></a>                                                                       In order to make the electric vehicle safe charging, the International Electro technical Committee (IEC) defined a set of standards, covering devices for protection and electrical installation standards, especially for residual current protection.

In IEC61851-1-2017 ( Electric vehicle conductive charging system –Part 1: General requirements ) E3.0
In IEC62752 – 2018 ( In-cable control and protection device for mode 2 charging of electric road vehicles (IC-CPD) ) E1.1                                                                                                                                         In IEC62955 - 2018 ( Residual direct current detecting device (RDC-DD) to be used for mode 3 charging of electric vehicles ) E1.0                                                                                                                   <a href="https://sm.ms/image/6k8vpeFw7DhnKRi" target="_blank"><img src="https://i.loli.net/2021/11/17/6k8vpeFw7DhnKRi.jpg" ></a>                                                             After studied the three IEC Standards, we found that TYPE A Residual Current plus A Smooth DC Residual Current Protection is requested.

Magtron – All type Current Sensitive Residual Current monitoring Unit (RCMU Series Sensor) both for AC and DC.

Magtron RCMU Series Residual Current sensor specifically developed for EV-Charging Providing all-current sensitivity. The sensor can trigger an automatic shut-off (Trip) signal in case of hazardous electrical faults ( DC and AC ) according to IEC 62752 or IEC 62955.

	Benefits:
	Protects against hazardous situations with fault currents, Preserving RCCB type A from saturation in case a smooth DC component in the circuit
	Small volume and footprint for integration into compact IC-CPD and AC Charging Wall-box and Charging Piles
	Robust mechanical and electrical design suitable for harsh environmental conditions
	Electrical safety at low cost ( compared to TYPE B RCCB)
	Integrated self- check functions

	Applications (EV Charging Mode 2 and Mode 3)
	IC-CPD in charging Cables for electric vehicles (EV) and plug-in hybrid electric vehicles (PHEV).
	AC Charging Wall-box for EV and PHEV.
	AC Charging piles for EV and PHEV

	Specifications
	SM Series
<a href="https://sm.ms/image/hk5lcPezfOb3vRM" target="_blank"><img src="https://i.loli.net/2021/11/17/hk5lcPezfOb3vRM.jpg" ></a>                                                                    <a href="https://sm.ms/image/1yj4Ew5NZGqoUYl" target="_blank"><img src="https://i.loli.net/2021/11/17/1yj4Ew5NZGqoUYl.png" ></a>                                                                    	SN-MINI and SN Series                                                                                                                               <a href="https://sm.ms/image/4u2zmqgDtWJ7xyp" target="_blank"><img src="https://i.loli.net/2021/11/17/4u2zmqgDtWJ7xyp.jpg" ></a>                                                                 
<a href="https://sm.ms/image/qSIHQKy3gCmeahA" target="_blank"><img src="https://i.loli.net/2021/11/17/qSIHQKy3gCmeahA.png" ></a>                                                                 	Certification
	TUV SUV Mark: B 1135980001
	TUV Rheinland: 505140000001                                                                                                                   	DEKRA Attestation Of Conformity : 6097030.01                                                                                   <a href="https://sm.ms/image/BAkf7FTMHcqhIrl" target="_blank"><img src="https://i.loli.net/2021/11/17/BAkf7FTMHcqhIrl.jpg" ></a>                                                                       	Reference Applications Cases
Magtron RCMU has been widely used in European standard EV charging mode 2 (IC-CPD) and mode 3 (AC Charging Pile and Wallbox)
	EV charging mode 2 (IC-CPD) 
<a href="https://sm.ms/image/Ybksp94LQE5CFgu" target="_blank"><img src="https://i.loli.net/2021/11/17/Ybksp94LQE5CFgu.jpg" ></a>                                                                    	EV charging mode 3 (AC Charging Pile and Wallbox)                                                                       <a href="https://sm.ms/image/sWfgqTKEQkuvonX" target="_blank"><img src="https://i.loli.net/2021/11/17/sWfgqTKEQkuvonX.jpg" ></a>                                                                    
	If you need more product information, PLS contact bellow:
	E-mail: morgan.wang@magtron.com.cn
	Website:  www.magtronworld.com
	Tel: +86-573-82660100

**Comment by anburocky3** [comment]
Is it related to `Tailwind CSS`? 😟

**Reply by NovusOpus** [reply]
I think it's some complex new way of Phishing, it's trying to pass a redirect link over an image link and image URL text

**Comment by NovusOpus** [comment]
@adamwathan @simonswiss @RobinMalfait Seems like a new complex way of Phishing.

There's an outside link attached to every image and inside Image URL text.

Btw is this the right way of reporting Phishing or should we report them via discord or any other way?  instead of mentioning your user names with @ ?
